packages/docs/docs/app/admin-page/admin-page.md

```diff
@@ -0,0 +1,35 @@
+# The Admin Page
+
+The [Admin Page](https://app.medplum.com/admin/project) of the Medplum App allows admin users to view and edit details of their project that normal users do not have access to.
+
+At the top of the page, there is an array of tabs as shown below. In this guide, we will briefly go over the content of each tab as well as any functionality they provide.
+
+![App Admin Page](./admin-page.png)
+
+## Details
+
+The Details tab displays all of the populated elements of the current [`Project`](/docs/api/fhir/medplum/project) resource. For more details on the [`Project`](/docs/api/fhir/medplum/project) resource, see the [User Management Guide](/docs/auth/user-management-guide).
+
+## Users
+
+The Users tab displays all of the [`Practitioner`](/docs/api/fhir/resources/practitioner) resources that are also a [`User`](/docs/api/fhir/medplum/user) in your project. It also allows you to invite new users. For more details on the [`User`](/docs/api/fhir/medplum/user) resource, see the [User Management Guide](/docs/auth/user-management-guide).
+
+## Patients
+
+The Patients tab displays all of the [`Patient`](/docs/api/fhir/resources/patient) resources that are also a [`User`](/docs/api/fhir/medplum/user) in your project. It also allows you to invite new patients.
```

Comment:
Nit: Like the section above, add a pointer to the User Management Guide. Specifically, to the section about project-scoped users